crates/holochain_sqlite/src/db/conn.rs

```diff
@@ -0,0 +1,64 @@
+use crate::prelude::*;
+use rusqlite::*;
+use std::ops::{Deref, DerefMut};
+
+pub type PConnInner = r2d2::PooledConnection<r2d2_sqlite::SqliteConnectionManager>;
```

Comment:
if this is only exposed in `PConn` it can have the same visibility

```suggestion
pub(super) type PConnInner = r2d2::PooledConnection<r2d2_sqlite::SqliteConnectionManager>;
```

Reply:
Oh yes, good spot.